src/components/StoriesList/Pagination.tsx

```diff
@@ -0,0 +1,68 @@
+"use client";
+
+import { usePathname, useSearchParams } from "next/navigation";
+
+interface Props {
+  total_pages: number;
+}
+
+export default function Pagination({ total_pages }: Props) {
+  const params = Object.fromEntries(useSearchParams().entries());
+  if (!params.page) {
+    params.page = "1";
+  }
+  const page_number = parseInt(params.page);
+  const pages = Array.from({ length: total_pages }, (_, i) => i + 1);
+
+  return (
+    <>
+      <ul className="inline-flex justify-center gap-4 align-middle text-lg text-sciquelMuted">
+        <li>Page</li>
```

Comment:
I think this can probably be removed. Having numbers at the bottom of the page implies that this is the page number.

Reply:
I actually pulled this from Edward's design so @edwrd-chen are you comfortable I remove "Page" from pagination here?

![Screenshot 2023-07-15 at 6 39 30 PM](https://github.com/SciQuel/SciQuel/assets/25441447/a67fe3fc-81be-4a2f-9f1a-ba23e0d225de)